packages/framework/build.xml

```diff
@@ -653,29 +653,28 @@
         <touch file="${path.root}/MAINTENANCE"/>
     </target>
 
-    <target name="npm" description="Installs modules required for Grunt.">
-        <exec executable="${path.npm.executable}" dir="${path.root}" logoutput="true" passthru="true" checkreturn="true">
-            <arg value="prune"/>
-            <arg value="--no-audit"/>
-        </exec>
+    <target name="npm" description="Build node modules.">
+        <phingcall target="npm-install-dependencies"/>
 
         <exec executable="${path.npm.executable}" dir="${path.root}" logoutput="true" passthru="true" checkreturn="true">
-            <arg value="install"/>
+            <arg value="run"/>
+            <arg value="trans"/>
         </exec>
 
         <exec executable="${path.npm.executable}" dir="${path.root}" logoutput="true" passthru="true" checkreturn="true">
             <arg value="run"/>
-            <arg value="copyFw"/>
+            <arg value="build"/>
         </exec>
+    </target>
 
+    <target name="npm-install-dependencies" description="Installs node modules dependencies.">
```

Comment:
_nitpick_
Can be this target used by itself? (eg. without running `trans` and `build`)

If it's unlikely, I suggest hiding this target (argument ` hidden="true"` as it is in phplint for example)

Reply:
Ok, fixed